docs/01-app/04-api-reference/04-functions/use-link-status.mdx

```diff
@@ -0,0 +1,202 @@
+---
+title: useLinkStatus
+description: API Reference for the useLinkStatus hook.
+---
+
+`useLinkStatus` is a **Client Component** hook that lets you track the loading state of a `Link` component during navigation. It can be used to show loading indicators during page transitions, especially when [prefetching](/docs/app/building-your-application/routing/linking-and-navigating#2-prefetching) is disabled, or the routed page does not have a suspense boundary.
```

Comment:
Nit

```suggestion
`useLinkStatus` is a **Client Component** hook that lets you track the loading state of a `Link` component during navigation. It can be used to show loading indicators during page transitions, especially when [prefetching](/docs/app/building-your-application/routing/linking-and-navigating#2-prefetching) is disabled, or the linked route does not have any loading states. 
```